### PR TITLE
[ES] Add/Update `People` articles

### PR DESCRIPTION
**Pull request by agatemosu**
- [x] Requires #8556 and #8557

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

`SKIP_WIKILINK_CHECK`

### CODE REVIEW

File: wiki/People/Tournament_Committee/es.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## ¿Cómo puedo ayudar a contribuir?
+
+¡Puedes ayudar al comité del torneo al continuar organizando excelentes torneos! Si tiene algún problema o inquietud con respecto a cualquier aspecto de un torneo de la comunidad, lo alentamos a que nos lo comunique a través del formulario de informes del torneo.
+
+Aquellos que estén interesados en unirse al comité de torneos pueden expresarnos su interés enviándonos un correo electrónico a [tournaments@ppy.sh](mailto:tournaments@ppy.sh). Se espera que los candidatos exitosos tengan una amplia experiencia en la escena del torneo, ya sea como jugador o personal.
+
+## Miembros
+
+- ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578)
+- ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251)
```
Comment (juankristal) [inline wiki/People/Tournament_Committee/es.md:22]: Add `D I O` as he was recently added in a previous PR.

File: wiki/People/Users_with_unique_titles/es.md [hunk 2]
```diff
@@ -251,62 +245,63 @@ Son [Artistas destacados](/wiki/Featured_Artists) con osu! y sus cuentas son pre
...
+Estos jugadores recibieron sus respectivos títulos a través de un logro de juego loco o al completar un desafío específico.
 
-| Usuario | Titulo | Logro |
+| Usuario | Título | Logro |
 | :-- | :-- | :-- |
 | ::{ flag=NO }:: [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | Consiguió el primer NoMod FC en [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
-| ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Consiguió hacer FCs en todos los mapas rankeados de osu!taiko a la fecha del 3 de noviembre del 2019 |
-| ::{ flag=NL }:: [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | *Sin documentar* <!-- TODO: a reference to TWEWY's 104 building, because he got to rank #104 in a certain amt of time.. need to investigate further --> |
-| ::{ flag=US }:: [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Jugo todos los mapas rankeados de osu! a la fecha del 27 de Abril del 2019 |
+| ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Consiguió FC en todos los mapas clasificados de osu! taiko a partir del 3 de noviembre de 2019 |
```
Comment (agatemosu) [inline wiki/People/Users_with_unique_titles/es.md:289]:
```suggestion
| ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Consiguió FC en todos los mapas clasificados de osu! taiko a partir del 3 de noviembre de 2019 |
``

Comment (agatemosu) [inline wiki/People/Users_with_unique_titles/es.md:289]:
```suggestion
| ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Consiguió FC en todos los mapas clasificados de osu!taiko a partir del 3 de noviembre de 2019 |
```

**Comment by juankristal** [comment]
Will check this after I am back from a family trip at around the 5th of January. Though I believe it would be best to split future translations like this one into smaller ones that are faster to check.

For example, having one PR for t-committee, one for CCs, and so on.

**Comment by agatemosu** [comment]
I will move some translations to another PR

File: wiki/People/The_Team/es.md [hunk 3]
```diff
@@ -1,87 +1,93 @@
...
 
 | Nombre | Descripción |
 | :-- | :-- |
-| [Beatmap Nominators](Beatmap_Nominators) | Usuarios moviendo cielo y tierra para asegurarse que tus mapas sean calificados |
-| [Developers](Developers) | Haciendo que el juego sea impresionante al agregar nuevas funciones y corregir los errores |
-| [Global Moderation Team](Global_Moderation_Team) | Encargados de vigilar los foros y el chat del juego |
+| [Beatmap Nominators](Beatmap_Nominators) | Usuarios que van más allá de la llamada para asegurarse de que sus mapas sean calificados |
+| [Beatmap Spotlight Curators](Beatmap_Spotlight_Curators) | Responsables de seleccionar mapas de alta calidad para [beatmaps destacados](/wiki/Beatmap_Spotlights) |
+| [Desarrolladores](Developers) | Haciendo que el juego sea increíble agregando nuevas funciones y corrigiendo los errores |
+| [Equipo de Moderación Global](Global_Moderation_Team) | Vigilar los foros y el chat del juego |
```
Comment (agatemosu) [inline wiki/People/The_Team/es.md:55]:
```suggestion
| [Global Moderation Team](Global_Moderation_Team) | Vigilar los foros y el chat del juego |
```

File: wiki/People/The_Team/es.md [hunk 4]
```diff
@@ -1,87 +1,93 @@
...
 | [Nomination Assessment Team](Nomination_Assessment_Team) | Administrar a los Beatmap Nominators |
-| [osu! Alumni](osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
-| [Project Loved Team](Project_Loved_Team) | Reconociendo los mapas que más ama la comunidad |
-| [Support Team](Support_Team) | Ayuda / Asistencia al usuario |
+| [osu! Alumni](osu!_Alumni) | Aquellos conocidos por sus contribuciones que desde entonces se han mudado |
+| [Equipo del Project Loved](Project_Loved_Team) | Reconocer los beatmaps que más le gustan a la comunidad |
```
Comment (agatemosu) [inline wiki/People/The_Team/es.md:58]:
```suggestion
| [Project Loved Team](Project_Loved_Team) | Reconocer los beatmaps que más le gustan a la comunidad |
```

**Review by juankristal — APPROVED** [review]

**Pull request merged**